docs/source/examples/combine_ddd_zne.md

```diff
@@ -0,0 +1,241 @@
+
+print(circuit)
+```
+
+In the diagram above, the horizontal brakets above and below specific gates represent the [cirq](https://quantumai.google/cirq) notion of [moments](https://quantumai.google/reference/python/cirq/Moment), which are time-slices intended to help with qubit scheduling (e.g. making sure qubits arrive at a gate at the same time). 
+
+In the our noise model, we will apply errors after each moment. Adding the additional identity gates within each "moment" should amplify the effect of these time-correlated errors.
+
+**Note:** Due to cirq's definition of moments, the first two gates of the circuit (`cirq.H(qubits[0])` and `cirq.CNOT(qubits[0], qubits[i])`) are in the same moment. This means the idle steps cannot be added between these gates, and therefore the time correlated noise will have less of an effect for the start of the circuit. 
+
++++
+
+## Noise model and executor
+
+**Importantly**, since DDD is designed to mitigate time-correlated (non-Markovian) noise, we simulate systematic $R_z$ rotations and depolarising noise applied to each qubit after each time step. This corresponds to noise which is strongly time-correlated and, therefore, likely to be mitigated by DDD. We 
+
+We use an [executor function](../guide/executors.md) to run the quantum circuit with the noise model applied.
+
+```{code-cell}
+def execute(
+    circuit: cirq.Circuit, 
+    rz_noise: float = 0.01,
+    depolar_noise: float = 0.0005
+    ) -> MeasurementResult:
+    """
+    Execute a circuit with R_z dephasing noise of strength ``rz_noise`` and depolarizing noise ``depolar_noise``
+    """
+    # Simulate systematic dephasing (coherent RZ) on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.rz(rz_noise))
+
+    # Simulate systematic depolarizing on each qubit for each moment.
+    circuit = circuit.with_noise(cirq.bit_flip(depolar_noise))
+
+    # Measure out all qubits
+    circuit += cirq.measure(*sorted(circuit.all_qubits()), key="m")
+
+    # Use a noise simulator
+    simulator = cirq.DensityMatrixSimulator()
+
+    # Run the circuit 1000 times
+    result = simulator.run(circuit, repetitions=1000)
+
+    # Retrieve the measurement results in bitstring form
+    bitstrings = result.measurements["m"]
+
+    return MeasurementResult(bitstrings)
+```
+
+Let's see what the execute function returns when we call it on our GHZ circuit, just leaving the default noise levels:
+
+```{code-cell}
+res = execute(circuit)
+res.to_dict() # Dictionary for more convenient visualization
+```
+
+The `'counts'` key represents the number of times a particular measurement bitstring occurred. 
+
+Since the default execute function includes noise in this case, we see that most of the results are all ones or all zeros, corresponding to perfect entanglement between all qubits.
+
+A few others, like `'110111'` however are one or more bitflips away from either of these states. It is these types of errors we want to mitigate. 
```

Comment:
I like the level of details in this explanation!

Reply:
Thanks!